src/index.js

```diff
@@ -0,0 +1,34 @@
+import './style.css';
+
+const list = document.querySelector('.list');
+
+function displayLists(toDo) {
+  const text = `<li class="item">
+                       <i class="uil uil-square circle-icon" job="complete"></i>
+                       <p class="text" > ${toDo}<p>
+                       <i class="uil uil-trash-alt" job="delete"></i>
+               </li>`;
+
+  const position = 'beforeend';
+  list.insertAdjacentHTML(position, text);
+}
+const arr = [];
+let index = 0;
+
+document.addEventListener('keyup', (event) => {
```

Comment:
* On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task. currently, your project lacks this functionality kindly fix this 🙏